src/librustdoc/clean/auto_trait.rs

```diff
@@ -75,6 +80,33 @@ impl<'a, 'tcx, 'rcx> AutoTraitFinder<'a, 'tcx, 'rcx> {
         self.get_auto_trait_impls(did, &def_ctor, Some(name))
     }
 
+    fn get_real_ty<F>(&self, def_id: DefId, def_ctor: &F, real_name: &Option<Ident>,
```

Comment:
nit: this function signature should follow rustfmt block indent style (especially since it looks like the function below already follows that